spring-ws-core/src/main/java/org/springframework/ws/transport/http/HttpComponents5MessageSender.java

```diff
@@ -209,15 +216,16 @@ protected HttpContext createContext(URI uri) {
 	@Override
 	public void destroy() throws Exception {
 
-		if (getHttpClient()instanceof CloseableHttpClient client) {
+		if (getHttpClient() instanceof CloseableHttpClient client) {
 			client.close();
```

Comment:
Unfortunately, this is a flaw in IntelliJ IDEA's Eclipse Format plugin when the LHS ends in a function invocation. Nevertheless, we'll keep things as-is to minimize manual editing.

Reply:
I reverted the effect of the spring-javaformat-maven-plugin.